src/app/category/[category]/page.tsx

```diff
@@ -2,7 +2,7 @@
 
 import React from "react";
 import MyPageTitle from "@/components/molecules/title/my-page-title";
-import MyFormCategory from "@/components/molecules/my-form-category/my-form-category";
+import MyFormCategory from "@/components/molecules/myFormCategory/myFormCategory";
```

Comment:
I don't get why you changed the component my-form-category ?

Reply:
demande de sébastien :/

Reply:
can't you re-rename it ? just so we don't get conflicts in the end